frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -160,6 +165,16 @@ export default {
     }
     await this.getOrganization()
     this.loadedApplications = await ApplicationService.getActiveApplications()
+
+    //get the redirected applications so we can prevent those facilities from starting another OFM app
+    this.userInfo?.facilities
```

Comment:
This code is very similar to what's in ApplicationsHistoryView. Could we simplify it by passing in a list of facilities (or active applications even) to the service and have it return an array of redirected applications? I.e. query all redirected applications for facilities which don't have an active applicaiton.

Reply:
yes - done :)